uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/contract-entitlements.html

```diff
@@ -0,0 +1,31 @@
+<div class="panel panel-default">
+  <div class="panel-heading">
+    <h3 class="panel-title">Contract Entitlements</h3>
+  </div>
+  <div class="table-responsive" ng-show="entitlements.contractsLoaded && entitlements.contracts.length > 0">
+    <table class="table">
+      <thead>
+        <tr>
+          <th style="width:140px">Position</th>
+          <th>Start Date</th>
+          <th>End Date</th>
+          <th ng-repeat="absenceType in entitlements.absenceTypes">{{absenceType.title}}</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr ng-repeat="contract in entitlements.contracts">
+          <td>{{contract.position}}</td>
+          <td>{{contract.start_date}}</td>
+          <td>{{contract.end_date}}</td>
+          <td ng-repeat="absence in contract.absences">{{absence.amount}}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+  <div class="panel-body" ng-show="entitlements.contractsLoaded && entitlements.contracts.length === 0">
+    None found.
+  </div>
+  <div class="panel-body" ng-show="!entitlements.contractsLoaded">
+    <crm-loading></crm-loading>
```

Comment:
Done. However, we should discuss the layout (grey loading background instead of white with @AkA84 )